jupyterlab_hdf/util.py

```diff
@@ -18,11 +20,23 @@ def dsetChunk(dset, row, col):
     return dset[slice(*row), slice(*col)].tolist()
 
 
+def serialize_value(v):
+    """
+    Turns a value into a JSON serializable version
+    """
+    if isinstance(v, list):
+        return [serialize_value(i) for i in v]
+    if isinstance(v, np.ndarray):
+        return serialize_value(v.tolist())
```

Comment:
Why are you re-serializing the array after it's been converted to `list`? According to [the docs](https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.tolist.html#numpy-ndarray-tolist, `tolist` gets applied recursively automatically.

Reply:
So that the bytes are turned into strings.